enterprise-suite/scripts/lbc.py

```diff
@@ -338,22 +338,54 @@ def pvs_retained_and_not(namespace):
                  '{{ printf "%s %s %s %s\\n" .metadata.name .spec.persistentVolumeReclaimPolicy .spec.claimRef.name .status.phase }}{{ end }}{{ end }}'
                 )
     returncode, stdout, _ = run("kubectl get pv -o go-template='{}'"
-                                .format(go_template))
+                                .format(go_template), show_stderr=False)
+    # pvc-8149517b-3bae-11e9-83b7-08002755d2dd Delete alertmanager-storage Bound
```

Comment:
What's this comment?

Reply:
That and the comment at line 356 show what is expected from running the command.  I could remove or add an `# output like` comment...